src/harness/fourslash.ts

```diff
@@ -584,18 +584,23 @@ namespace FourSlash {
 
         public verifyGoToDefinition(arg0: any, endMarkerNames?: string | string[]) {
             this.verifyGoToX(arg0, endMarkerNames, () => this.getGoToDefinition());

```

Comment:
not sure why we need two calls here. `getGoToDefinitionAndBoundSpan` subsumes the other one, so would be sufficient to do  here.

Reply:
i would say always use `getGoToDefinitionAndBoundSpan` and check the span only if the marker name was provided.

Reply:
Changed as suggested.